code/obj/item/bowling.dm

```diff
@@ -72,6 +85,18 @@
 							src.hitWeak(hitMob, user)
 		return
 
+	proc/pre_attack_damage_modifier(obj/item/parent_item, atom/A, mob/user)
+		var/mob/living/carbon/human/human_user = user
+		if(human_user.w_uniform && istype(human_user.w_uniform, /obj/item/clothing/under/gimmick/bowling))
+			//bashing someones skull in with a bowling ball should hurt if you are worthy of the bowling ball
+			src.force = 20
+			src.stamina_damage = 40
+			spawn(1)
```

Comment:
This is extremely cursed, couldn't you just do
```
attackby()
	src.force = x
	..()
	src.force = initial(src.force)
```
and not need the spawn or signal registration boilerplate? (this isn't a great solution either, but it's at least a bit better)